bindings/gumjs/gumquickcore.c

```diff
@@ -3622,9 +3660,31 @@ gum_quick_native_callback_invoke (ffi_cif * cif,
   GumInvocationContext * ic;
   GumQuickInvocationContext * jic = NULL;
   JSValue this_obj;
+  GumQuickCallbackContext * jcc = NULL;
   int argc, i;
   JSValue * argv;
   JSValue result;
+  guintptr frame_pointer = 0, return_address = 0, stack_pointer = 0;
+
+#if defined(HAVE_ARM64)
+  asm ("mov %0, x29" : "=r" (frame_pointer));
+  asm ("mov %0, x30" : "=r" (return_address));
+  asm ("mov %0, sp" : "=r" (stack_pointer));
+
+  return_address &= G_GUINT64_CONSTANT (0x7fffffffff);
+#elif defined(HAVE_ARM)
+  asm ("mov %0, r7" : "=r" (frame_pointer));
+  asm ("mov %0, lr" : "=r" (return_address));
+  asm ("mov %0, sp" : "=r" (stack_pointer));
+#elif defined(HAVE_I386)
+# if GLIB_SIZEOF_VOID_P == 8
+  asm ("movq %%rbp, %0" : "=m" (frame_pointer));
+  asm ("movq %%rsp, %0" : "=m" (stack_pointer));
+# else
+  asm ("mov %%ebp, %0" : "=m" (frame_pointer));
+  asm ("mov %%esp, %0" : "=m" (stack_pointer));
+# endif
```

Comment:
This needs a special-case for when `_MSC_VER` is defined (Windows). That might be a painful context-switch (unless you happen to have a Windows dev env standing by), and if so I can take a look at it once I've cleared the other items on the Frida 15 backlog. Would be great to get this into the upcoming release! 🔥

Reply:
i have no windows here 😞  , so if you could take a look that would be great 🤘 !

Reply:
This should be fixed in last commit